src/admin/__tests__/createAcls.spec.js

```diff
@@ -0,0 +1,228 @@
+const createAdmin = require('../index')
+// const { KafkaJSProtocolError } = require('../../errors')
+// const { createErrorFromCode } = require('../../protocol/error')
+
+const {
+  secureRandom,
+  createCluster,
+  newLogger,
+  sslConnectionOpts,
+  saslBrokers,
+} = require('testHelpers')
+
+const RESOURCE_TYPES = require('../../protocol/resourceTypes')
```

Comment:
See https://github.com/tulios/kafkajs/issues/649#issuecomment-597131598 for some issues regarding `RESOURCE_TYPES`.

Basically, we have been misusing ResourceTypes for configs, when they are supposed to be used in the way that you are here, for ACLs. So before we start having more people use ResourceTypes, I would like to figure out a good way to split the two so that we have `CONFIG_RESOURCE_TYPES` and `ACL_RESOURCE_TYPES`.

The old `RESOURCE_TYPES` still needs to be exported for backwards compatibility until we decide to release a major version, but it would be good if we can prepare by having most people already now import either CONFIG_RESOURCE_TYPES or ACL_RESOURCE_TYPES, so that the break affects fewer people.

So what I would suggest is that we within this PR introduce `ACL_RESOURCE_TYPES`, which is just the old `RESOURCE_TYPES` re-exported under a different name, and use that in all examples and tests.

Reply:
In fact, we should probably prefix all the ACL-related protocol types with `ACL_` just to avoid this in the future.

Reply:
@Nevon  SGTM, let's do that, also I would suggest to keep types in a separate folder like: `src/protocol/types/ACL/Resource.js` or `src/protocol/types/ACLResources.js`